### PR TITLE
bedrock: accept realm invite in getRealmFromInvite()

### PR DESCRIPTION
**Pull request by extremeheat**
cc @LucienHH

### CODE REVIEW

File: src/bedrock/api.js [hunk 1]
```diff
@@ -11,9 +11,14 @@ module.exports = class BedrockRealmAPI extends RealmAPI {
     return { host, port: Number(port) }
   }
 
+  async acceptRealmInvite (inviteCode) {
+    await this.rest.get(`/invites/v1/link/accept/${inviteCode}`)
```
Comment (LucienHH) [inline src/bedrock/api.js:15]: This should be a post request.

**Comment by LucienHH** [comment]
Currently, there are 2 ways of adding a Realm:
- By entering a Realm's invite code which is this endpoint 
- And by being invited by the host which the player has to accept in the menus. 

Handling both may mean we need to have separate functions
`acceptRealmInviteFromCode` - Handles Realm code invites
`acceptRealmInviteFromId` - Handles individual invites

What'd you think?

**Comment by extremeheat** [comment]
Do you want to do the PR?

**Comment by LucienHH** [comment]
https://github.com/PrismarineJS/prismarine-realms/pull/18 Could you check this?

**Pull request closed without merging**